models/ad_reporting__ad_group_report.sql

```diff
@@ -21,6 +21,13 @@ aggregated as (
         sum(clicks) as clicks,
         sum(impressions) as impressions,
         sum(spend) as spend
+
+        {%- if var('ad_reporting__passthrough_metrics') -%}
+            {% for metric in var('ad_reporting__passthrough_metrics') %}
+                , sum({{ metric }}) as {{ metric }}
```

Comment:
This ensures the name of the metric is leveraged.

```suggestion
                , sum({{ metric }}) as {{ metric.name }}
```

Reply:
Same comment for the other models

Reply:
Didn't know that, thanks! Changed 👍